openpype/hooks/pre_copy_last_published_workfile.py

```diff
@@ -31,16 +32,12 @@ def execute(self):
         2- Check if workfile in work area doesn't exist
         3- Check if published workfile exists and is copied locally in publish
         4- Substitute copied published workfile as first workfile
+           with incremented version by +1
 
         Returns:
             None: This is a void method.
         """
 
-        sync_server = self.modules_manager.get("sync_server")
```

Comment:
So if sync server is not available or enabled it will happen anyway?

Reply:
This part comes from https://github.com/pypeclub/OpenPype/pull/4102 (which this PR depends on) so I think it's more appropriate to discuss it in #4102
But to answer your question the check still exists it's just been moved to `download_last_published_workfile` in `sync_server.py`

Reply:
I think these kind of checks must remain in hooks to have explicit logging.